InventoryManagement/src/main/java/com/ccs/inventorymanagement/route/DeleteClothingHandler.java

```diff
@@ -0,0 +1,45 @@
+package com.ccs.inventorymanagement.route;
+
+import com.ccs.inventorymanagement.config.RouteConfig;
+import com.ccs.inventorymanagement.domain.Clothing;
+import com.ccs.inventorymanagement.domain.Item;
+import com.ccs.inventorymanagement.service.ClothingService;
+import lombok.Builder;
+import lombok.Data;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.reactive.function.BodyInserters;
+import org.springframework.web.reactive.function.server.HandlerFunction;
+import org.springframework.web.reactive.function.server.ServerRequest;
+import org.springframework.web.reactive.function.server.ServerResponse;
+import reactor.core.publisher.Mono;
+
+import java.util.UUID;
+import java.util.function.Function;
+
+public class DeleteClothingHandler implements HandlerFunction<ServerResponse> {
+
+    private final ClothingService service;
+
+    public DeleteClothingHandler(ClothingService service) {
+        this.service = service;
+    }
+
+    @Override
+    public Mono<ServerResponse> handle(ServerRequest serverRequest) {
+        return serverRequest.bodyToMono(UpdateClothingHandler.Request.class)
```

Comment:
Was using the UpdateClothingHandler.Request here intentional?

Reply:
Yeah, I think we talked about this already where since they use the same Request class we can do this right? If that changes in the future, we can add a Request class to the delete handler as well but for right now I don't see a reason to have redundant code?